Doc/using/cmdline.rst

```diff
@@ -563,23 +563,22 @@ Miscellaneous options
    It also allows passing arbitrary values and retrieving them through the
    :data:`sys._xoptions` dictionary.
 
-   .. versionchanged:: 3.2
-      The :option:`-X` option was added.
+   .. versionadded:: 3.2
 
-   .. versionadded:: 3.3
-      The ``-X faulthandler`` option.
+   .. versionchanged:: 3.3
```

Comment:
These are all additions of new flags... "changed" implies they existed before and their behavior has changed.

When the text starts with "Added the" to me that _implies_ it should be versionadded not versionchanged?  At a glance I don't see this covered in our documentation style guide.

_Per the issue you filed you are defining things in a consistent manner, just not what I assumed it'd be._

Reply:
This is a temporary change, later I plan to add the `versionadded` directives just under the description of every `-X xxx` option. But this change looks larger and deserves a separate PR. 

If you prefer, I'll remove these changes and create a separate PR for a different approach. But if keep all versionmodified directives in one place, I think that they should be `versionchanged` because they are related to the change in the `-X` option as a whole.

Reply:
See #116304.

Reply:
Lets just go with this PR.  It moves us towards more consistency regardless of the specifics.